docs/management/admin/host-isolation-exceptions.asciidoc

```diff
@@ -0,0 +1,41 @@
+[[host-isolation-exceptions]]
+[chapter]
+= Host isolation exceptions
+
+You can configure host isolation exceptions for specific IP addresses that <<host-isolation-ov, isolated hosts>> are still allowed to communicate with after they've been blocked from the rest of your network. Isolated hosts can still send data to {es} and {kib}, so you don't need to set up host isolation exceptions for them.
+
+Host isolation exceptions support IPv4 addresses, with optional CIDR (classless inter-domain routing) notation.
+
+IMPORTANT: Each host isolation exception IP address should be a highly trusted and secure location, since you’re allowing it to communicate with hosts that you've isolated due to suspicious activity.
```

Comment:
Another point to make here is that users isolate hosts to stop the potential spread of a threat.  So any IP they except could still be at risk.

You do mention that it's a `secure location`, so maybe that's enough.  Was just wondering if there was a more up front way to make that risk of a threat spreading more clear.

Reply:
Good point! How about something like this -- does this go far enough to make the risk clear?
```suggestion
IMPORTANT: Each host isolation exception IP address should be a highly trusted and secure location, since you’re allowing it to communicate with hosts that you've isolated to contain a potential threat and prevent it from spreading.
```

Reply:
looks good!